src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1480,6 +1480,8 @@ private void getURLs(String baseUrl, SiteNode parent, ApiResponseList list) {
 		map.put("solution", alert.getSolution());
 		if (alert.getHistoryRef() != null) {
 			map.put("messageId", String.valueOf(alert.getHistoryRef().getHistoryId()));
+		} else {
+			map.put("messageId", "");
```

Comment:
Could use the ternary operator like in the setters (it would avoid the duplication of the name).